x-pack/plugins/security_solution/public/common/mock/endpoint/app_context_render.tsx

```diff
@@ -166,26 +167,56 @@ export const createAppRootMockRenderer = (): AppContextTestRender => {
   };
 };
 
-const createCoreStartMock = (): ReturnType<typeof coreMock.createStart> => {
+const createCoreStartMock = (
```

Comment:
I found that I need the same fix for unit tests in my PR, but I don't have the change in `management`, so I just do some mock to pass the test. Please be ware of this when merging this.